packages/docs/source/components/tab.md

```diff
@@ -0,0 +1,306 @@
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
+    <div class="ods-tabs" id="example-0" style="
+      --ods-indicator-width: 99px;
+      --ods-indicator-pos-x: 0;
+    ">
+      <div class="ods-tablist" role="tablist" aria-label="">
+        <button class="ods-tab" id="tablistitem-0-0" role="tab" aria-selected="true" aria-controls="tab-0-0">Agents</button>
+        <button class="ods-tab" id="tablistitem-0-1" role="tab" aria-selected="false" aria-controls="tab-0-1">People</button>
+        <button class="ods-tab" id="tablistitem-0-2" role="tab" aria-selected="false" aria-controls="tab-0-2">Settings</button>
+        <button class="ods-tab" id="tablistitem-0-2" role="tab" aria-selected="false" aria-controls="tab-0-3">Import</button>
+      </div>
+      <div class="ods-tabpanel">
+        <div id="tab-0-0" tabindex="0" role="tabpanel" aria-labelledby="tablistit-0-0">
+          <p>Tab Panel 0: Agents</p>
+        </div>
+        <div id="tab-0-1" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-0-1" hidden="">
+          <p>Tab Panel 1: People</p>
+        </div>
+        <div id="tab-0-2" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-0-2" hidden="">
+          <p>Tab Panel 2: Settings</p>
+        </div>
+        <div id="tab-0-3" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-0-3" hidden="">
+          <p>Tab Panel 3: Import</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</figure>
+
+```html
+    <div class="ods-tabs" id="example-0">
+      <div class="ods-tablist" role="tablist" aria-label="">
+        <button class="ods-tab" id="tablistitem-0-0" role="tab" aria-selected="false" aria-controls="tab-0-0">
+          Agents
+        </button>
+        <button class="ods-tab" id="tablistitem-0-1" role="tab" aria-selected="false" aria-controls="tab-0-1">
+          People
+        </button>
+        <button class="ods-tab" id="tablistitem-0-2" role="tab" aria-selected="false" aria-controls="tab-0-2">
+          Settings
+        </button>
+        <button class="ods-tab" id="tablistitem-0-2" role="tab" aria-selected="false" aria-controls="tab-0-3">
+          Import
+        </button>
+      </div>
+      <div class="ods-tabpanel">
+        <div id="tab-0-0" tabindex="0" role="tabpanel" aria-labelledby="tablistit-0-0">
+          <p>Tab Panel 0: Agents</p>
+        </div>
+        <div id="tab-0-1" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-0-1" hidden="">
+          <p>Tab Panel 1: People</p>
+        </div>
+        <div id="tab-0-2" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-0-2" hidden="">
+          <p>Tab Panel 2: Settings</p>
+        </div>
+        <div id="tab-0-3" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-0-3" hidden="">
+          <p>Tab Panel 3: Import</p>
+        </div>
+      </div>
+    </div>
+  </div>
+```
+## Anatomy
```

Comment:
I'm unsure how helpful this section is without either a visual breakdown or a code breakdown. If we want to document which classes are in play and what they do / what they represent, that could be useful to Eng, but I think we'd want to move it down the page a bit.